src/metabase/driver.clj

```diff
@@ -248,7 +248,7 @@
   driver has difference escaping rules for table or schema names when used from metadata.
 
   For example, oracle treats slashes differently when querying versus when used with `.getTables` or `.getColumns`"
-  {:arglists '([driver table-name])}
+  {:arglists '([driver table-name]), :added "0.37.0"}
```

Comment:
Unrelated but I added this extra bit of metadata because I was trying to remember when this method go added while looking into this bug fix. Doesn't hurt to have it.